packages/flutter_tools/test/general.shard/vmservice_test.dart

```diff
@@ -459,6 +459,14 @@ void main() {
           method: 'getVMTimeline',
           errorCode: RPCErrorCodes.kServiceDisappeared,
         ),
+        const FakeVmServiceRequest(
```

Comment:
Can you also add a successful test where the stats are logged?

Reply:
Done!